react/features/facial-recognition/functions.js

```diff
@@ -131,8 +157,69 @@ export async function sendDataToWorker(
         return;
     }
 
+    if (typeof OffscreenCanvas === 'undefined') {
+        const canvas = document.createElement('canvas');
```

Comment:
I realize I missed this during review last time. I think we should generate a single canvas and reuse it. This way we are creating a ton of DOM nodes which may end up causing problems due to GC.